Wikipedia/Categories/UIImage+WMFStyle.m

```diff
@@ -14,4 +14,15 @@ + (UIImage*)wmf_placeholderImage {
     return img;
 }
 
++ (instancetype)wmf_imageFromColor:(UIColor*)color {
+    CGRect rect = CGRectMake(0, 0, 1, 1);
```

Comment:
if this needs to stay, can we cache these? no need to recreate the same image a bunch of times, and this kind of stuff can be expensive.